docs/architecture/adr-050-sign-mode-textual.md

```diff
@@ -0,0 +1,492 @@
+# ADR 050: SIGN_MODE_TEXTUAL
+
+## Changelog
+
+- Dec 06, 2021: Initial Draft.
+- Feb 07, 2022: Draft read and concept-ACKed by the Ledger team.
+
+## Status
+
+Draft. Concept ACK by the Ledger team, but details need to be polished.
+
+## Abstract
+
+This ADR specifies SIGN_MODE_TEXTUAL, a new string-based sign mode that is targetted at signing with hardware devices.
+
+## Context
+
+Protobuf-based SIGN_MODE_DIRECT was introduced in [ADR-020](./adr-020-protobuf-transaction-encoding.md) and is intended to replace SIGN_MODE_LEGACY_AMINO_JSON in most situations, such as mobile wallets and CLI keyrings. However, the [Ledger](https://www.ledger.com/) hardware wallet is still using SIGN_MODE_LEGACY_AMINO_JSON for displaying the sign bytes to the user. Hardware wallets cannot transition to SIGN_MODE_DIRECT as:
+
+- SIGN_MODE_DIRECT is binary-based and thus not suitable for display to end-users,
+- hardware cannot decode the protobuf sign bytes due to memory constraints.
+
+In an effort to remove Amino from the SDK, a new sign mode needs to be created for hardware devices. [Initial discussions](https://github.com/cosmos/cosmos-sdk/issues/6513) propose a string-based sign mode, which this ADR formally specifies.
+
+## Decision
+
+We propose to have SIGN_MODE_TEXTUAL’s signing payload `SignDocTextual` to be an array of strings. Each string corresponds to one "screen" on the hardware wallet device, with no (or little) additional formatting done by the hardware wallet itself.
```

Comment:
what are the pros/cons of using a list of strings, rather than one long string?

Reply:
Clarified. We're signing a `\n`-delimited string.